app/views/posts/show.html.erb

```diff
@@ -1,2 +1,75 @@
-<h1>Posts#show</h1>
-<p>This is going to show all details from one post of one user</p>
\ No newline at end of file
+<section class="pagination">
```

Comment:
Post show page is not displaying post with all comments listed as shown on the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_single_post.png?raw=true). PLease fix this, I read your text on the `Additional Information` and in order for you to add Posts without having a view, you can use the rails console.

- To create a post without a view you can use the rails console, you can check this [link](https://guides.rubyonrails.org/command_line.html) to the rails guide to have more info about it.